pkg/services/featuremgmt/toggles_gen_test.go

```diff
@@ -55,6 +55,9 @@ func TestFeatureToggleFiles(t *testing.T) {
 			if flag.Owner == "" {
 				t.Errorf("feature %s does not have an owner. please fill the FeatureFlag.Owner property", flag.Name)
 			}
+			if flag.Stage == FeatureStageGeneralAvailability && flag.Expression == "" {
+				t.Errorf("feature %s should be explicitly enabled or disabled if it's in GA. please add Expression property", flag.Name)
+			}
```

Comment:
Could it make sense to also check that `Expression` is either `"true"` or `"false"`? I can see myself making a typo and write `"treu"` for instance.

Reply:
Good point. I think it makes sense. IIUC anything else then "true" resolves to false.

Reply:
Added additional validation in 224e16e9be9e5d7b9f939c9fee062ace21e361e7